Examples/Physics_applications/laser_acceleration/PICMI_inputs_laser_acceleration.py

```diff
@@ -102,7 +102,8 @@
 
 field_diag1 = picmi.FieldDiagnostic(name = 'diag1',
                                     grid = grid,
-                                    period = 10)
+                                    period = 1,
```

Comment:
Thanks for making this change!

This should be `period = 10` to match the value in ParticleDiagnostic.

Reply:
Oops indeed, thanks